tensorflow/workspace.bzl

```diff
@@ -730,6 +720,18 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
         },
     )
 
+    # Intel openMP that is part of LLVM sources.
+    tf_http_archive(
+        name = "llvm-openmp",
```

Comment:
Got an internal error prohibiting hyphens in package name. Would you mind renaming this to `llvm_openmp` instead? Thank you!

Reply:
@penpornk made the changes. Thanks.